scripts/genpinctrl/genpinctrl.py

```diff
@@ -505,6 +501,12 @@ def main(cube_path, output):
                         if af["name"] not in entries:
                             entries[af["name"]] = list()
 
+                        if family != "STM32F1":
```

Comment:
`mode` from config file should take precedence over inferred mode. So I think we should have:

```python
if not af.get("mode") and family != "STM32F1":
```

Reply:
Indeed, fixed